### ISSUE TITLE
Semantically rename the wrapper `<div>` 

### ISSUE THREAD

**Issue by csantiago132** [KyleAMathews/react-headroom#128]
Hi, great plugin btw. I have a sugeestion/idea/request

right now, this plugin renders 

```html
<div class="headroom-wrapper" > ... </div>
```

I would love to rename it by either props or whatever way you prefer to 

```html 
<header class="headroom-wrapper" > ... </div>
```

in react, maybe add a ` tag` prop so we can semantically rename the component if needed?
something like

```js
<Headroom
  tag="header"
  ...
>
```

Thanks again for the great job and hope this gets implemented someday

**Comment by Ozymandias9000** [comment]
Looks like there's a passing PR open for this: https://github.com/KyleAMathews/react-headroom/pull/162

**Comment by janczizikow** [comment]
Should be solved with v3.1.0. Thanks to @dbilgin it's now possible to change the HTML element which will get rendered 🙂

**Comment by dbilgin** [comment]
Thanks for taking care of the PR @janczizikow
